src/sentry/db/models/fields/node.py

```diff
@@ -118,7 +120,7 @@ def data(self):
             rv = self.field.wrapper(rv)
         return rv
 
-    def bind_data(self, data, ref=None):
+    def bind_data(self, data, ref=None, event_datetime=None):
```

Comment:
This method is also called 8 lines above, can't we have the same issue there ?

Reply:
We can have the same issue there, just wanted to do a simple implementation until we determine this is a legit issue.